doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -109,7 +109,9 @@ Please make sure to set your preferred storage location ***before*** migration.
   - Removed `dontCheckEnabled` config support from Areablock editable. 
   - Functionality that was moved into its own bundle and must be enabled manually in `config/bundles.php`
     - Glossary has been moved into PimcoreGlossaryBundle
+    - Xliff Translation has been moved into PimcoreXliffBundle
```

Comment:
```suggestion
    - Xliff Translations Export/Import has been moved into PimcoreXliffBundle
```

Reply:
Should we also mention moved events here?

Reply:
Probably would need a generic line for all bundles, but added anyways one for this specific bundle for an heads-up